### ISSUE TITLE
bug: Image container lambda are not triggered by custom command

### ISSUE THREAD

**Issue by fmaronnaud** [localstack/localstack#5006]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Defining a custom command for an image container lambda does not invoke the lambda with that custom command.

### Expected Behavior

If a custom command is defined in the serverless template or the CFN template, the image container lambda should be invoked with that custom command.

### How are you starting LocalStack?

With a docker-compose file

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

docker-compose.yml:
```
version: "3.8"
services:
  localstack:
    container_name: "${LOCALSTACK_DOCKER_NAME-localstack_main}"
    image: localstack/localstack
    network_mode: bridge
    ports:
      - "127.0.0.1:53:53"                
      - "127.0.0.1:53:53/udp"          
      - "127.0.0.1:443:443"             
      - "127.0.0.1:4510-4530:4510-4530"  
      - "127.0.0.1:4566:4566"
      - "127.0.0.1:4571:4571"
    environment:
      - SERVICES=${SERVICES- }
      - DEBUG=${DEBUG- }
      - DATA_DIR=${DATA_DIR- }
      - LAMBDA_EXECUTOR=docker     
      - LOCALSTACK_API_KEY=XXXXXXXXX
      - DOCKER_HOST=unix:///var/run/docker.sock
      - HOST_TMP_FOLDER=C:\lstmp
      - DOCKER_FLAGS=1
    volumes:
      - 'C:\lstmp:/tmp/localstack'
      - '/var/run/docker.sock:/var/run/docker.sock'
  
volumes:
  localstack-vol:
```

###### Image container lambda:
```
{
	"AWSTemplateFormatVersion": "2010-09-09",
	"Description": "The AWS CloudFormation template for this Serverless application",
	"Resources": {
		"EventProcessing": {
			"Type": "AWS::IAM::Role",
			"Properties": {
				"RoleName": "foo-event-processing-local",
				"AssumeRolePolicyDocument": {
					"Statement": [{
						"Effect": "Allow",
						"Principal": {
							"Service": ["lambda.amazonaws.com"]
						},
						"Action": ["sts:AssumeRole"]
					}]
				},
				"Policies": [{
					"PolicyName": "foo-event-processing-local-policy",
					"PolicyDocument": {
						"Version": "2012-10-17",
						"Statement": [{
							"Effect": "Allow",
							"Action": ["logs:CreateLogGroup", "logs:CreateLogStream", "logs:PutLogEvents"],
							"Resource": "*"
						}]
					}
				}]
			}
		}
	},
	"Outputs": {
		"ServerlessDeploymentBucketName": {
			"Value": "foo-deploymentbucket-local"
		},
		"EventProcessingRoleArn": {
			"Description": "ARN of new event processing role",
			"Value": {
				"Fn::GetAtt": ["EventProcessing", "Arn"]
			},
			"Export": {
				"Name": "EventProcessingRoleArn"
			}
		}
	},
	"StackName": "foo-event-processing-local-role"
}
```
```
{
	"AWSTemplateFormatVersion": "2010-09-09",
	"Description": "The AWS CloudFormation template for this Serverless application",
	"Resources": {
		"MainLogGroup": {
			"Type": "AWS::Logs::LogGroup",
			"Properties": {
				"LogGroupName": "/aws/lambda/foo-event-processing-local-main"
			}
		},
		"MainLambdaFunction": {
			"Type": "AWS::Lambda::Function",
			"Properties": {
				"Code": {
					"ImageUri": "localhost:4511/serverless-foo-event-processing-local@sha256:7f23f81d0bbb6e2e1a0283bc7a90bb9098aee4324a7137c9dbde293744a19f18"
				},
				"ImageConfig": {
					"Command": ["EventProcessing::EventProcessing.Handlers::Main"]
				},
				"PackageType": "Image",
				"FunctionName": "foo-event-processing-local-main",
				"MemorySize": 1024,
				"Timeout": 6,
				"Role": "arn:aws:iam::000000000000:role/foo-event-processing-local"
			},
			"DependsOn": ["MainLogGroup"]
		},
		"MainLambdaVersion6gkQd7Yg6Kazrff4lE5yBzF09mzObMK2U5Dj9IPexHU": {
			"Type": "AWS::Lambda::Version",
			"DeletionPolicy": "Retain",
			"Properties": {
				"FunctionName": {
					"Ref": "MainLambdaFunction"
				},
				"CodeSha256": "7f23f81d0bbb6e2e1a0283bc7a90bb9098aee4324a7137c9dbde293744a19f18"
			}
		}
	},
	"Outputs": {
		"ServerlessDeploymentBucketName": {
			"Value": "foo-deploymentbucket-local"
		},
		"MainLambdaFunctionQualifiedArn": {
			"Description": "Current Lambda function version",
			"Value": {
				"Ref": "MainLambdaVersion6gkQd7Yg6Kazrff4lE5yBzF09mzObMK2U5Dj9IPexHU"
			},
			"Export": {
				"Name": "sls-foo-event-processing-local-MainLambdaFunctionQualifiedArn"
			}
		}
	},
	"StackName": "foo-event-processing-local-code"
}
```

Note that in this case, we will be expecting the image to be run with the command `EventProcessing::EventProcessing.Handlers::Main`

### Environment

```markdown
- OS: Windows 10 
- LocalStack version: 0.13.0
- Serverless Framework Core: 2.66.2
- Serverless Plugin: 5.5.1
- Serverless SDK: 4.3.0
- Serverless Components: 3.18.1
- Serverless-localstack: ^0.4.35
```

##### Docker image used by the lambda:

```
ARG AWS_LAMBDA_VERSION=local

FROM aws-lambda-dotnet:$AWS_LAMBDA_VERSION AS base
FROM mcr.microsoft.com/dotnet/sdk:5.0-buster-slim as build

ARG SERVICE_DIR
ARG PROJECT_NAME

COPY services/$SERVICE_DIR /$PROJECT_NAME
WORKDIR /$PROJECT_NAME

RUN dotnet build "$PROJECT_NAME.csproj" -c Docker -o /app/build

FROM build AS publish
RUN dotnet publish "$PROJECT_NAME.csproj" -c Docker -o /app/publish

FROM base AS final
WORKDIR /var/task
COPY --from=publish /app/publish .
```

**Comment by whummer** [comment]
Thanks for reporting @fmaronnaud . This should be (hopefully) fixed in the meantime via #5007 . Can you please pull the `latest` Docker image and give it another try? Please let us know if the problem persists.. Thanks!

**Comment by fmaronnaud** [comment]
Thanks for the quick turnaround and confirming this is now working as expected while using Localstack 0.13.0.7.
Good job folks
